### PR TITLE
Makes the clear emoji happen with the edit of the reaction

### PR DESCRIPTION
**Pull request by yannickgloster**

### CODE REVIEW

File: cogs/csgo.py [hunk 1]
```diff
@@ -198,7 +206,7 @@ async def pug(self, ctx):
                 'players': team2_steamIDs
             },
             'cvars': {
-                'get5_event_api_url': f'http://{self.bot.web_server.IP}:{self.bot.web_server.port}/'
+                'get5_event_api_url': f'http://83.197.169.148:{self.bot.web_server.port}/'
```
Comment (lexesj) [inline cogs/csgo.py]: Hardcoded IP

File: cogs/csgo.py [hunk 2]
```diff
@@ -148,11 +154,13 @@ async def pug(self, ctx):
 
             player_veto_count += 1
 
+
         message_text = 'Map Veto Loading'
         players_text = 'None'
         embed = self.player_veto_embed(message_text=message_text, players_text=players_text, team1=team1,
                                        team1_captain=team1_captain, team2=team2, team2_captain=team2_captain)
         await message.edit(content=message_text, embed=embed)
+        message.clear_reactions()
```
Comment (lexesj) [inline cogs/csgo.py]: Needs `await`

**Review by lexesj — CHANGES_REQUESTED** [review]

File: cogs/csgo.py (hunk 2 above)
Comment (yannickgloster) [reply cogs/csgo.py]: Fixed

File: cogs/csgo.py (hunk 1 above)
Comment (yannickgloster) [reply cogs/csgo.py]: Fixed IP

**Pull request merged**